modules/deployment-model-impl/src/test/java/org/mule/runtime/module/deployment/impl/internal/builder/ApplicationFileBuilder.java

```diff
@@ -148,7 +151,7 @@ public ApplicationFileBuilder containingPlugin(ArtifactPluginFileBuilder plugin)
   public ApplicationFileBuilder sharingLibrary(String jarFile) {
     checkImmutable();
     checkArgument(!StringUtils.isEmpty(jarFile), "Jar file cannot be empty");
-    resources.add(new ZipResource(jarFile, "plugins/lib/" + FilenameUtils.getName(jarFile)));
+    resources.add(new ZipResource(jarFile, LIB_FOLDER + separator + SHARED_FOLDER + separator + getName(jarFile)));
```

Comment:
Can we have both methods? One for getting the relative path and another one for getting the absolute path?.